src/Worker/CreateShadowentry.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * @file src/Worker/CreateShadowentry.php
+ * @brief This script creates posts with UID = 0 for a given public post.
+ *
+ * This script is started from mod/item.php to save some time when doing a post.
+ */
+
+namespace Friendica\Worker;
+
+require_once("include/threads.php");
+
+class CreateShadowentry {
```

Comment:
Can you please rename it to `CreateShadowEntry` please?

Reply:
I guess this is some German compound wording :-)